frontend.rst

```diff
@@ -33,6 +33,13 @@ to solve the most common Webpack use cases.
 
 .. _encore-toc:
 
+Technical Requirements
+----------------------
+
+* Node.js on your development machine(s)
+* Yarn or npm on your development machine(s)
+* No requirements on your production machine(s)
```

Comment:
It depends, you can either build localy or not

Reply:
Yeah, but "No requirements on your production machine" is still true, since it's not *required* to install it there. But it's certainly *allowed* :-) See the "show-stopper" argument above.